src/plugins/data/public/index_patterns/utils.ts

```diff
@@ -51,10 +51,10 @@ export async function findByTitle(
 
 export function getRoutes() {
   return {
-    edit: '/management/kibana/index_patterns/{{id}}',
```

Comment:
@ppisljar
It's a little bit strange, why two different plugins (`data` and `indexPatternManagement`) are responsible for routing (keep routes map). 
I've started to research it and didn't find where we use it. 
From my point of view `indexPatternsService` should  not be responsible for that.  

I've tried to remove it here: #65683 . Waiting CI......

Any ideas?

Reply:
i agree alex, thanks for noting this, it looks like its not used